client/my-sites/theme/main.jsx

```diff
@@ -676,11 +678,11 @@ class ThemeSheet extends Component {
 
 		const plansUrl = siteSlug ? `/plans/${ siteSlug }/?plan=value_bundle` : '/plans';
 
-		const { canonicalUrl, description, name: themeName } = this.props;
+		const { canonicalUrl, description, seoDescription, name: themeName, seoTitle } = this.props;
```

Comment:
Continuing from the previous comment, these 2 are undefined and we should expect to find them under `seo_description` and `seo_title`.